src/status_im/contexts/wallet/common/sheets/remove_account/view.cljs

```diff
@@ -64,10 +68,13 @@
             :checked?            @confirmed?
             :on-change           #(swap! confirmed? not)}]
           [quo/text (i18n/label :t/remove-account-confirmation)]]
-         [footer {:submit-disabled? (not @confirmed?)}]]))))
+         [footer
+          {:submit-disabled? (not @confirmed?)
+           :address          address
+           :toast            (i18n/label :t/account-removed)}]]))))
```

Comment:
Will the `footer` dispatch a toast? :thinking:

Reply:
It's a toast message after successful deletion

Reply:
Replaced with `:toast-message`